### DISCUSSION TITLE
Does the minified js contain XMLBuilder?

### DISCUSSION THREAD

**Discussion by shimona-usds** [NaturalIntelligence/fast-xml-parser#687] [Q&A]
I would like to rebuild some xml i parsed earlier in a plugin so as to view it. Since it's a plugin I'm limited so I was hoping to include the fxparser.min.js ? Does it include the XMLBuilder and how do I call it?

**Comment by amitguptagwl** [comment]
[webpack config](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/webpack-prod.config.js) can provide you this detail.Eg
- fxp.min.js: parser, validator, builder
- fxbuilder.min.js: builder only
- fxparser.min.js: parser only
- fxvalidator.min.js: validator only
